tests/test_real_projects.py

```diff
@@ -53,11 +52,11 @@ def verify_requirements(venv_path: Path, requirements: List[str]) -> None:
 def run_fawltydeps_json(
     *args: str, venv_dir: Optional[Path], cwd: Optional[Path] = None
 ) -> JsonData:
-    cmd = ["fawltydeps"]
-    if venv_dir:
-        cmd = [f"{venv_dir}/bin/fawltydeps"]
+    argv = ["fawltydeps", "--config-file=/dev/null", "--json"]
```

Comment:
Is this supposed to be `args`?

Reply:
It's fine either way so feel free to do either and resolve. It's the first time that I see `argv` so I'll let you judge whether that's known in the Python community.